2-library/domain-model-6core-java/src/main/java/com/_6core/lib/java/domain/immutable/cart/ShoppingCartV01Impl.java

```diff
@@ -0,0 +1,17 @@
+package com._6core.lib.java.domain.immutable.cart;
```

Comment:
when you have package `service` with interfaces, and you are going to implement its you create the `service/impl` package. 

We strive to get rid of  too generous words like `service` or `impl` and replace with meaningful names. 
So... let's create `cart/immutable` package structure. 
because CART is represent cart domain model. 
and IMMUTABLE is represent the type of implementation of CART. 

in case any other implementation, e.g. `mutable` or `record` or `proto` and so on. We just create a new group of `impl` with proper name.